CHANGELOG.md

```diff
@@ -41,6 +45,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Processings which are not the result of joinery are now serialized with `TimberElement`.
 * Fixed visualization bug in `Plate` due to loft resulting in flipped volume.
 * Fixed a few bugs in the `WallPopulator` workflow including GH component updates.
+* Changed how input `Polyline`s are handled in `Plate`.
```

Comment:
what's the meaning of the change for a user? if public interface changed somehow than perhaps mention it here.